src/boot/efi/splash.c

```diff
@@ -135,28 +135,51 @@ static EFI_STATUS bmp_parse_header(
         return EFI_SUCCESS;
 }
 
-static void pixel_blend(uint32_t *dst, const uint32_t source) {
-        uint32_t alpha, src, src_rb, src_g, dst_rb, dst_g, rb, g;
+enum Channels { R, G, B, A, _CHANNELS_MAX };
```

Comment:
hmm, you name this enum "Channels", but never actually make use of the type name. Might as well make this anonymous...

I find "R" a *very* short identifier. Maybe at least call this CHANNEL_R? and similar for the others...